<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="QA1073" name="com.leafBot.testcases.TC001_LoginLogOut" tests="1" failures="0" timestamp="2022-10-31T10:01:23 IST" time="9.903" errors="1">
  <testcase name="createLeaf" time="9.903" classname="com.leafBot.testcases.TC001_LoginLogOut">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method com.leafBot.testcases.TC001_LoginLogOut.createLeaf() didn&#039;t finish within the time-out 2000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method com.leafBot.testcases.TC001_LoginLogOut.createLeaf() didn't finish within the time-out 2000
at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNoExecutor(MethodInvocationHelper.java:348)
at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:281)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:604)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- createLeaf -->
  <system-out/>
</testsuite> <!-- com.leafBot.testcases.TC001_LoginLogOut -->
